api-2.2/src/test/java/org/openmrs/module/htmlformentry/EncounterDiangnosesTagHandlerComponentTest.java

```diff
@@ -0,0 +1,113 @@
+package org.openmrs.module.htmlformentry;
```

Comment:
Typo in file name ... Diagnoses not Diangnoses

Reply:
Cool, thanks for this catch @mogoodrich.